modules/ROOT/pages/rest-sdk/edits/features.adoc

```diff
@@ -0,0 +1,69 @@
+= Overview
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+REST SDK provides metadata-driven features that create a unified connectivity
+experience for all types of users:
+
+* As an API developer, use REST SDK to produce connectivity at scale.
+* As an integration specialist, use REST SDK to create connections when
+your desired connector is not available in Studio.
+* As an API product manager, use REST SDK to expand API outreach to developers
+and non-developer users.
+* As an API enthusiast, use REST SDK to develop and publish use-case based
+connections for a growing marketplace.
+
+== Features in REST SDK
+
+REST SDK currently supports the following features:
+
+* Instant connector generation from RAML 1.0, OAS 2.0, and OAS 3.0
+* Maven distribution framework
```

Comment:
```suggestion
* Apache Maven distribution framework
```
This is just something I know: The Apache Software Foundation tends to be a bit sticky about use of its trademarks, so it's safest to provide attribution of them when introducing them in text. If you mention Maven a lot, it's OK to do so without Apache, once that has been established.

Reply:
Good to know!